shared/utility/math/filter/YawFilter.hpp

```diff
@@ -60,9 +60,14 @@ namespace utility {
                 /// @brief Previous kinematic yaw for bias estimation
                 Scalar previous_kinematic_yaw = 0.0;
 
+                /// @brief Maximum allowed bias
+                Scalar max_bias = 0.1;
```

Comment:
```suggestion
                Scalar max_bias = 0.0;
```
probably just zero it by default and have the config set it?